PhysicsTools/PatAlgos/python/tools/jetTools.py

```diff
@@ -170,8 +170,8 @@ def setupSVClustering(btagInfo, svClustering, algo, rParam, fatJets=cms.InputTag
             btagInfo.groomedFatJets = groomedFatJets
 
 
-def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSource, elSource, muSource, runIVF, loadStdRecoBTag, svClustering, fatJets, groomedFatJets,
-                  algo, rParam, btagDiscriminators, btagInfos, patJets, labelName, postfix):
+def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSource, elSource, muSource, runIVF, hipMitigation, loadStdRecoBTag, svClustering, fatJets, groomedFatJets,
+                  algo, rParam, btagDiscriminators, btagInfos, patJets, labelName, btagPrefix, postfix):
```

Comment:
Yes, the prefix was necessary because with the current PATJetProducer code there is no other way to distinguish different instances of the same tagger. This is because in https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_17/PhysicsTools/PatAlgos/plugins/PATJetProducer.cc#L97-L104 everything after "JetTags" is stripped so there is no way to distinguish different instances of the same tagger based just on the postfix. So instead of messing around with the PATJetProducer code and possibly introducing backward incompatibilities, I think it was safer to add the prefix option.

Note that this feature will be useful more generally. When using `updateJetCollection` users will now be able to compare old and new version of the discriminator taking them from the same jet. Otherwise, the most recent discriminator will be picked up and to compare the two, one would have to deal with things like
```python
process.bDiscrAOD = cms.EDProducer("RecoJetDeltaRValueMapProducer",
    distMax = cms.double(0.4),
    matched = cms.InputTag("slimmedJets"),
    src = cms.InputTag("slimmedJets"),
    value = cms.string("bDiscriminator('pfCombinedInclusiveSecondaryVertexV2BJetTags')"),
    lazyParser = cms.bool(True)
)

updateJetCollection(
   process,
   jetSource = cms.InputTag('slimmedJets'),
   jetCorrections = ('AK4PFchs', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'None'),
   btagDiscriminators = ['pfCombinedInclusiveSecondaryVertexV2BJetTags']
)
process.updatedPatJets.userData.userFloats.src += ['bDiscrAOD']
```
Also, it is now possible to run multiple versions of the IVF sequence in parallel.